05week/checkers.js

```diff
@@ -60,7 +108,12 @@ function Game() {
 
   this.start = function() {
     this.board.createGrid();
-    // Your code here
+    this.board.createCheckers();
+    this.moveChecker = function(start, end) {
```

Comment:
`this.moveChecker` should not be nested inside `this.start`. This method is only called when a player makes a turn.